docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -231,7 +234,7 @@ and additional supplemental YAML file ``values2.yaml`` with content
        date: '{{ yyyymmdd }}'
        empty: null
        greeting: Hello
-       message: Hello World
+       message: 'Hello World '
```

Comment:
I think that `_reify_scalar_str()` (removed in a recent-past PR, but I didn't update the docs then) would have been removing the trailing space from a value like this. Without that second parse, the strings retain the space and so appear quoted to preserve it.